Writerside/topics/Moonbear-and-artifacts-overview.md

```diff
@@ -4,16 +4,17 @@
 
 Relic
 
-+1 Striking Silver Flesh gem/Battle Axe
++1 Striking Standanrt-Grade Silver Flesh Gem/Battle Axe
 
 You can switch between the two versions as a free action on your turn.
 
 If you drop or get disarmed of the axe it will go back into the Flesh Gem.
 
 The flesh gem is counted as a **Free-Hand** **Agile** weapon with a damage dice of **1d6** and is in the **brawling** group.
```

Comment:
Is it a free-hand weapon or an unarmed attack since its technically part of my body? (if it's an unarmed attack tho you can use it with monk features!!)

Reply:
> Standanrt

Misspelt!